src/actions/userManagement.js

```diff
@@ -152,3 +152,25 @@ export const updateUserFinalDayStatus = (user, status, finalDayDate) => {
     });
   };
 };
+
+export const updateUserFinalDayStatusu = (user, status, finalDayDate,isSet) => {
+  const userProfile = Object.assign({}, user);
+  userProfile.endDate = finalDayDate;
+  userProfile.isActive = status === "Active";
+  userProfile.isSet = isSet ==="FinalDay";
```

Comment:
Consider changing all `isSet` to `isFinalDaySet` to be more accurate. We might have more properties of `userProfile` to accord set or not set in the future.

Reply:
`isSet` for now not only be modified in this pr, but have already existed in the repo, I may need to create a new Sprint task for it to track the modification tasks. Since this change is only name change and does not have effect to this commit, will balance to new tasks.

Reply:
Understood.